pkg/executor/executortype/poolmgr/gpm.go

```diff
@@ -711,7 +711,7 @@ func (gpm *GenericPoolManager) NoActiveConnectionEventChecker(ctx context.Contex
 	defer close(stopper)
 
 	var wg wait.Group
-	for _, informer := range utils.GetInformerEventChecker(ctx, kubeClient, "WsConnectionStarted") {
+	for _, informer := range utils.GetInformerEventChecker(ctx, kubeClient, "NoActiveConnections") {
```

Comment:
Yes, changing the event to `NoActiveConnections`, seems reasonable. 
Did you test the changes locally?

Reply:
No.  It needs a k8s cluster to work
I'll build a image to test in my local cluster locally.

Reply:
I have test on my k3s cluster. It works. The pod still live in 2min.